examples/c-bsp-test/linker.lds

```diff
@@ -0,0 +1,299 @@
+/* Copyright (c) 2020 SiFive Inc. */
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Apache-2.0: retain the License notice but also clearly state below what was changed from the original.

Reply:
Does this look okay [3e9e41f](https://github.com/soc-hub-fi/headsail-vp/pull/13/commits/3e9e41f96ccce11bb8cb1f36e04a973b38eee66f)?

Reply:
Seems good enough